File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// PassantenfrequenzStadtStgallen SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class PassantenfrequenzStadtStgallenMakeContext
{
    public static function call(array $ctxmap, ?PassantenfrequenzStadtStgallenContext $basectx): PassantenfrequenzStadtStgallenContext
    {
        return new PassantenfrequenzStadtStgallenContext($ctxmap, $basectx);
    }
}

File: go/utility/make_context.go
package utility

import "github.com/voxgig-sdk/passantenfrequenz-stadt-stgallen-sdk/go/core"

func makeContextUtil(ctxmap map[string]any, basectx *core.Context) *core.Context {
	return core.NewContext(ctxmap, basectx)
}
